open-api/swagger.yaml

```diff
@@ -753,6 +753,25 @@ paths:
           description: 'json that specifies the constraint. Example: `{"type":"study_name", "studyId":"EHR"}` or
           `{"type":"concept","path":"\\Public Studies\\CLINICAL_TRIAL_HIGHDIM\\High Dimensional data\\Expression Lung\\"}`.'
           type: string
+        - name: sort
+          required: false
+          in: query
+          type: string
+          description: >
+              json string that specifies the sort order of the observations. Sorting can be done on multiple
+              dimensions. `["patient"]` sorts the observations on the patient dimension, `["patient", "concept"]`
+              sorts the observations first on the patient, and then on concept. The sort order is on the 'key'
+              field of the chosen dimension, sorting on arbitrary fields is not yet supported. For patients, this is
+              the `id`, for concepts the `conceptCode`.
+
+              It is also possible to specify the sort order to be ascending or descending. Use `[['patient', 'asc'],
+              ['concept', 'desc']]` for that to sort the observations on patients first, ascending, and then on
+              concepts descending.
```

Comment:
it may be a bit confusing that for multiple dimensions the args format is different, so it's a list of lists, not [[PATIENT: 'desc'], [CONCEPT: 'asc']], but this shouldn't be a big problem. Maybe it is useful to add an example of the order specification for a single dimension, since it's different?

Reply:
True, the problem is that json doesn't have ordered maps, and this needs to be ordered. There are examples of the simpler syntax in the first paragraph.